examples/vi.rs

```diff
@@ -0,0 +1,132 @@
+use std::borrow::Cow::{self, Borrowed, Owned};
```

Comment:
You should have keep your example minimalist (like https://github.com/kkawakam/rustyline/blob/master/examples/read_password.rs)